<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:rgb(145, 190, 170);
}</string>
   </property>
   <widget class="QPushButton" name="bkonfirmasi">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>520</y>
      <width>280</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:rgb(0, 197, 145);
color: white;
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Konfirmasi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nama_balita">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Nama Balita</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>456</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;; color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="kenaikan_tinggi_badan">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Tinggi Badan (cm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Umur Balita</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kenaikan_berat_badan">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,7);
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Kenaikan Berat Badan (kg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>840</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:1px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>840</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>770</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Data Bulanan Balita</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tinggi_badan">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Berat Badan (kg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Kenaikan Tinggi Badan (cm)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="berat_badan">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,7);
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="umur_balita">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 12pt &quot;MS Shell Dlg 2&quot;;
padding-left:17px;
border-radius:1px;
border: 1px solid black

</string>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>bkonfirmasi</zorder>
   <zorder>nama_balita</zorder>
   <zorder>label_3</zorder>
   <zorder>error</zorder>
   <zorder>kenaikan_tinggi_badan</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>kenaikan_berat_badan</zorder>
   <zorder>label_8</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>tinggi_badan</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>berat_badan</zorder>
   <zorder>umur_balita</zorder>
  </widget>
 </widget>
 <resources>
  <include location="placeholder.qrc"/>
 </resources>
 <connections/>
</ui>
